### ISSUE TITLE
render all helper elements while DnD, an event w/ multiple resourceIds

### ISSUE THREAD

**Issue by kferrio** [fullcalendar/fullcalendar-scheduler#155]
### Feature request

We're evaluating Scheduler and really like what we see so far. We're wondering the following is on the roadmap:
#### Current state

JSBIN example here: http://jsbin.com/zekeku/edit?js,output
When dragging an `event` having two or multiple `resourceIds`, only the "sibling" under the mouse is rendered as a "ghost". When the dragged sibling is dropped, only it is rendered in a new location; its siblings are not rendered and are "lost". It is easy enough to POST on the drop and re-render. But the user loses the sense of where everything will drop and has to wait for the server to respond.
#### Future state

Please consider adding an option to 
1. "ghost" all siblings of a multi-resource event, during dragging of any of sibling of the set
2. upon drop, render all siblings in their new locations
#### Considerations
- Shifting a multi-resource event horizontally (in time) is a common use-case. Desired behavior: Horizontal drag moves the selected sibling and all of its siblings.
- But shifting a multi-resource event vertically (between resources in an ordered list) seems highly unlikely. 
- More likely is swapping one resource for another. So being able to drag one sibling vertically while leaving the other sibling instances in place would be very useful. Desired behavior: vertical drag moves the selected sibling only, leaving its siblings as they were. This is basically how Scheduler works now, except that the "stationary" siblings get "lost".
- Ideally, eventOverlap would fire during horizontal drags if _any_ of the siblings overalpped. Note that this problem is roughly _O(n)_ and not _O(n^2)_ because each sibling need be checked only  against its own timeline and not against the timelines of all resources.

**Comment by alan1o** [comment]
I think this is the same issue I raised here: 

https://github.com/fullcalendar/fullcalendar-scheduler/issues/111

**Comment by kferrio** [comment]
I think the two issues are related by common observations of current state.
I do not know if both issues are describing the same future state UX
behavior. It would be great if both could be closed with one commit.

**Comment by arshaw** [comment]
Yes, the meat of the problem you describe is represented in #111

But I will repurpose this ticket to represent the multiple visual ghost events that you'd like all rendered at the same time while dragging. That is a bit harder given the current state of the code, and will likely be implemented after #111.

**Comment by kferrio** [comment]
Thanks for looking at this.

**Comment by rob3c** [comment]
Hi, related to this is event resizing. It currently has similar behavior where you can only see the event under the mouse changing size while dragging, and the siblings are lost as soon as the resize starts. Do I need to create a separate issue for this, or is it implicitly covered by this issue and/or #111?

**Comment by arshaw** [comment]
@rob3c issue #111 is more appropriate for what you describe

**Comment by arshaw** [comment]
@rob3c could you please 👍  this issue?

**Comment by rob3c** [comment]
@arshaw done!

**Comment by headshine** [comment]
Current work around is to capture the eventDrop and reset the resourceIds and rerender the event. 

The same thing does not work for eventResizeStop though.

Looking forward to the fix!

**Comment by arshaw** [comment]
I accidentally fixed this in core v3.5.0 / scheduler 1.7.0

[See the blog post](https://fullcalendar.io/blog/2017/08/bootstrap-support-and-future-developments/)

demo: https://codepen.io/arshaw/pen/YxdoPK?editors=0010
